website/src/index.js

```diff
@@ -1,14 +1,19 @@
 import "./style.scss";
 import "mapbox-gl/dist/mapbox-gl.css";
 import Navigo from "navigo";
-import { injectPageBeforeRender, renderPages } from "./utils/utils";
+import {
+    injectPageBeforeRender,
+    updateInjectedPage,
+    renderInjectedPages,
```

Comment:
Couldn't it have been replace '*' though? - since it pretty much seems to cover all those methods from the file unless you really want to be namely specific about what you're importing.

Reply:
If only it was this easy. It's possible, though using an alias is kinda unwieldy, when the functions are repeated as often as they are throughout this file.